### PR TITLE
Added sorting by average rating

### PR DESCRIPTION
**Pull request by AnupKumarPanwar**
Fixes 
Make skill sorting by average stars rating #767 

Changes: 

Sorting parameters :
- filter_type=rating
- filter_name=ascending | descending

```
Sample endpoint

/cms/getSkillList.json?model=general&group=All&language=en&applyFilter=true&filter_name=descending&filter_type=rating
```

Screenshots for the change: 
image

### CODE REVIEW

**Comment by AnupKumarPanwar** [comment]
@Akshat-Jain That's varies from IDE to IDE. Some IDEs auto-indent the code.

**Comment by Akshat-Jain** [comment]
@AnupKumarPanwar I think your IDE is clearing any trailing spaces on saving files. Look for the default behaviour of trailing spaces in your IDE. You'll find the correct option there. Thanks :-)

**Comment by Orbiter** [comment]
arr... @AnupKumarPanwar please resolve conflicts

**Comment by AnupKumarPanwar** [comment]
@Orbiter Can it be merged now?

File: src/ai/susi/server/api/cms/ListSkillService.java [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import ai.susi.json.JsonObjectWithDefault;
 import ai.susi.mind.SusiSkill;
 import ai.susi.server.*;
+import com.sun.org.apache.xpath.internal.operations.Bool;
```
Comment (sudheesh001) [inline src/ai/susi/server/api/cms/ListSkillService.java]: This looks like it isn't being used at all. Could you please remove this import dependency?
Once this is done, I believe it's good to merge.

**Comment by AnupKumarPanwar** [comment]
@sudheesh001 Please review, the unused imports have been removed

**Pull request merged**